cfg-install

```diff
@@ -1,15 +1,20 @@
+#!/bin/sh
+
 git clone --bare https://github.com/munjal/dotfiles.git $HOME/.cfg
 
-function config {
-   /usr/bin/git --git-dir=$HOME/.cfg/ --work-tree=$HOME $@  
+git_mod()
```

Comment:
The main dot files has a function called *config*. I understand you want to have a better name of the function. I'll have to make changes in the script and billy etc. Let's keep this as it is for the the time being.